README.md

```diff
@@ -398,6 +399,9 @@ The following query parameters are supported
 - `receiveName`, only applies when `type` is `update` and is mandatory in this case. Specifies the smart contract receive name of the smart contract receive function being called.
 - `parameter`, only applies when `type` is `update` and is mandatory in this case. Specifies the smart contract parameter passed to the receive function. Given as a hex string.
 - `executionNRGBuffer`, optionally and only applies when `type` is `update`. Specifies a buffer in percentage of how much of the energy for invoking the contract should be added to the returned cost. Given as an   integer. If not provided, a default of 20% is used.
+- `tokenId`, the tokenId of the plt token in the plt token update transaction. The query parameter `tokenId` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
+- `listOperationsSize`, the size of the cbor encoded list of plt operations in the plt token update transaction. The query parameter `listOperationsSize` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
```

Comment:
```suggestion
- `tokenId`, the tokenId of the PLT in the PLT update transaction. The query parameter `tokenId` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
- `listOperationsSize`, the size of the cbor encoded list of PLT operations in the PLT update transaction. The query parameter `listOperationsSize` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
```

"PLT token" is redundant, since PLT = Protocol-Level Token.